CHANGELOG.md

```diff
@@ -81,6 +82,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - Fixed an issue on Windows where TeXworks path was not resolved if it was installed with MiKTeX. [#10977](https://github.com/JabRef/jabref/issues/10977)
 - We fixed an issue with where JabRef would throw an error when using MathSciNet search, as it was unable to parse the fetched JSON coreectly. [10996](https://github.com/JabRef/jabref/issues/10996)
 - We fixed an issue where the "Import by ID" function would throw an error when a DOI that contains URL-encoded characters was entered. [#10648](https://github.com/JabRef/jabref/issues/10648)
+- We fixed a bug where a new Systematic Literature Review was allowed to start even if a non-empty directory is selected, later throwing an error. [#620](https://github.com/koppor/jabref/issues/620)
```

Comment:
I think, that can be removed. Already included (somehow) in the other CHANGELOG.md entry above.

Reply:
Yes, initially I had thought since there was both a feature addition and a bug fix, I mentioned them separately in their respective sections. I'll remove this.